pkg/scripts/download.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Downloader downloads the specified URL to the target file location
 type Downloader interface {
-	DownloadFile(url *url.URL, targetFile string) (bool, error)
+	Download(url *url.URL, target string) (bool, error)
```

Comment:
I still don't like returing an bool from Download. What is the point of doing that? If the download was successful then the err is nil. If not, then err should contain details about what failed. We should get rid of using bools as a return value checks  and use it only in methods where it make sense.